main.go

```diff
@@ -188,12 +189,15 @@ func prune(cli *client.Client, deathNote *sync.Map) {
 			log.Println(err)
 		} else {
 			for _, container := range containers {
-				cli.ContainerRemove(context.Background(), container.ID, types.ContainerRemoveOptions{RemoveVolumes: true, Force: true})
+				err := cli.ContainerRemove(context.Background(), container.ID, types.ContainerRemoveOptions{RemoveVolumes: true, Force: true})
+				if err != nil {
+					log.Println(err)
+				}
 				deletedContainers[container.ID] = true
 			}
 		}
 
-		try.Do(func(attempt int) (bool, error) {
+		err = try.Do(func(attempt int) (bool, error) {
```

Comment:
A probably more robust approach would be extracting the try block to a function where it receives a closure. That way we could reuse it across the three different try blocks Wdyt?

Reply:
Agreed but I think that is out-of-scope for this pull request and could be done in a follow-up PR. thoughts?

Reply:
I agree, let's make sure this PR only focuses on the direct `go fmt` changes and linting errors.